### PR TITLE
re-enable the post_backdrop_filter test with CPU/GPU measurements

### PR DESCRIPTION
**Pull request by flar**

### CODE REVIEW

File: dev/devicelab/manifest.yaml [hunk 1]
```diff
@@ -549,11 +549,11 @@ tasks:
...
-  # post_backdrop_filter_perf_ios__timeline_summary:
-  #   description: >
-  #     Measures the runtime performance of animations after a backdrop filter is removed on iOS.
-  #   stage: devicelab_ios
-  #   required_agent_capabilities: ["mac/iphonexs"]
+   post_backdrop_filter_perf_ios__timeline_summary:
+     description: >
+       Measures the runtime performance of animations after a backdrop filter is removed on iOS.
+     stage: devicelab_ios
+     required_agent_capabilities: ["mac/iphonexs"]
```
Comment (liyuqian) [inline dev/devicelab/manifest.yaml:556]: I think we no longer have `iphonexs` in the devicelab. We probably should just use `mac/ios`.

Comment (liyuqian) [reply dev/devicelab/manifest.yaml:556]: Let's also add a `flaky: true` here just to be cautious.

Comment (flar) [reply dev/devicelab/manifest.yaml:556]: Done

**Comment by flar** [comment]
A question about "mac/ios" - does that mean that the test can be running on a variety of devices? Doesn't that cause the results to be inconsistent?

**Review by liyuqian — APPROVED** [review]
LGTM. `mac/ios` means iPhone 6s so it won't be running on other devices.

**Pull request merged**